src/jbuild

```diff
@@ -0,0 +1,17 @@
+(library
+ ((name        mirage_net)
+  (public_name mirage-net)
+  (libraries   (bytes result mirage-device))
+  (modules     (Mirage_net))
+  (flags       (:standard -w "A-4-41-44" -warn-error "+1..49" -safe-string))
```

Comment:
I'm not sure about adding warning errors to the build runes run after the release. I thing it's better to use `topkg build --dev` while doing development (which has a sane set of warning/error options).